spec/ParseUser.spec.js

```diff
@@ -3274,7 +3334,7 @@ describe('Parse.User testing', () => {
 
   it('should not allow updates to emailVerified', done => {
     const emailAdapter = {
-      sendVerificationEmail: () => {},
+      sendVerificationEmail: () => { },
```

Comment:
Where does this change come from? Can you revert these? Also others below.

Reply:
Unintended beautify. I removed that.